tululbot/commands.py

```diff
@@ -81,6 +95,11 @@ def _extract_slang_word(message):
         return message.text
 
 
+def _extract_birthday_boy_or_girl_name(message):
+    assert message.text is not None
+    return message.text[5:] if message.text.startswith('/hbd') else message.text
```

Comment:
Don't follow `_extract_leli_term` as an example. It is still buggy. You should follow `_extract_slang_word` instead by using regex to parse the command.